pkg/scheduler/scheduler.go

```diff
@@ -209,6 +213,7 @@ func (s *Scheduler) assign(ctx context.Context, e *entry) {
 	go func() {
 		err := s.client.Update(ctx, newWorkload)
 		if err == nil {
+			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Assigned", fmt.Sprintf("Successfully assigned to capacity %v", e.Capacity))
```

Comment:
successful sounds redundant, lets just say `Assigned to capacity %v`

Reply:
Done